app/mailers/application_mailer.rb

```diff
@@ -7,11 +7,19 @@ class ApplicationMailer < ActionMailer::Base
   helper :instance
   helper :formatting
 
+  after_action :set_autoreply_headers!
+
   protected
 
   def locale_for_account(account)
     I18n.with_locale(account.user_locale || I18n.default_locale) do
       yield
     end
   end
+
+  def set_autoreply_headers!
+    headers['Precedence'] = 'list'
+    headers['X-Auto-Response-Suppress'] = 'All'
+    headers['Auto-Submitted'] = 'auto-generated'
```

Comment:
I would maybe use just this one and not the others

Reply:
I agree that `Auto-Submitted` alone should be preferable if it works, but I'm not sure how different mail software handle it nor do I have ways to test it.